### PR TITLE
code normalization and fix PSR-2

### PR DESCRIPTION
**Pull request by lgcosta**

### CODE REVIEW

File: grid-map/src/table.php [hunk 1]
```diff
@@ -101,19 +100,24 @@
...
-              ($centreon->user->admin == 0 ? " AND T1.host_id = acl.host_id AND T2.host_id = acl.host_id AND acl.group_id IN (" .($grouplistStr != "" ? $grouplistStr : 0).")" : "").
-              " ORDER BY T1.name";
+    $query1 = "SELECT DISTINCT T1.name, T2.host_id
+        FROM hosts T1, hosts_hostgroups T2 " . ($centreon->user->admin == 0 ? ", centreon_acl acl " : "") . "
+        WHERE T1.host_id = T2.host_id 
+            AND T1.enabled = 1
+            AND T2.hostgroup_id = " . $preferences['host_group'] .
+                ($centreon->user->admin == 0
+                    ? " AND T1.host_id = acl.host_id AND T2.host_id = acl.host_id AND acl.group_id IN (" .
+                        ($grouplistStr != "" ? $grouplistStr : 0).")" : ""
```
Comment (sc979) [inline grid-map/src/table.php]:
```suggestion
                        ($grouplistStr != "" ? $grouplistStr : 0).")"
                    : ""
```

File: grid-map/src/table.php [hunk 2]
```diff
@@ -101,19 +100,24 @@
...
     /* Query 2 */
-    $query2 = "SELECT distinct T1.description ".
-              "FROM services T1 " .($centreon->user->admin == 0 ? ", centreon_acl acl " : "").
-              "WHERE T1.enabled = 1 ".
-              ($centreon->user->admin == 0 ? " AND T1.service_id = acl.service_id AND acl.group_id IN (" .($grouplistStr != "" ? $grouplistStr : 0).") AND (" : " AND (");
+    $query2 = "SELECT distinct T1.description
+        FROM services T1 " . ($centreon->user->admin == 0 ? ", centreon_acl acl " : "") . "
+        WHERE T1.enabled = 1 " . ($centreon->user->admin == 0
+            ? " AND T1.service_id = acl.service_id AND acl.group_id IN (" .
+                ($grouplistStr != "" ? $grouplistStr : 0).") AND (" : " AND ("
```
Comment (sc979) [inline grid-map/src/table.php]:
```suggestion
                ($grouplistStr != "" ? $grouplistStr : 0).") AND ("
            : " AND ("
```

File: grid-map/src/table.php [hunk 3]
```diff
@@ -125,11 +129,14 @@
...
-              "WHERE T1.enabled = 1 " .
-              "AND T1.description NOT LIKE 'ba_%' AND T1.description NOT LIKE 'meta_%' ".
-              ($centreon->user->admin == 0 ? " AND T1.service_id = acl.service_id AND acl.group_id IN (" .($grouplistStr != "" ? $grouplistStr : 0).")" : "");
+    $query3 = "SELECT DISTINCT T1.service_id, T1.description, T1.state, T1.host_id
+        FROM services T1 " . ($centreon->user->admin == 0 ? ", centreon_acl acl " : "") . "
+        WHERE T1.enabled = 1
+            AND T1.description NOT LIKE 'ba_%' AND T1.description NOT LIKE 'meta_%' " .
+                ($centreon->user->admin == 0
+                    ? " AND T1.service_id = acl.service_id AND acl.group_id IN (" .
+                    ($grouplistStr != "" ? $grouplistStr : 0).")" : ""
```
Comment (sc979) [inline grid-map/src/table.php]:
```suggestion
                        ($grouplistStr != "" ? $grouplistStr : 0).")"
                    : ""
```

**Comment by lgcosta** [comment]
thanks @sc979 !

**Pull request merged**